src/stream/src/common/table/state_table.rs

```diff
@@ -245,10 +248,17 @@ fn consistent_old_value_op(row_serde: impl ValueRowSerde) -> OpConsistencyLevel
                 true
             }
         }),
-        is_log_store: false,
+        is_log_store,
     }
 }
 
+#[derive(Eq, PartialEq, Copy, Clone, Debug)]
+pub enum StateTableOpConsistencyLevel {
+    Inconsistent,
+    ConsistentOldValue,
+    LogStoreEnabled,
+}
```

Comment:
It seems that these three levels are in a progressive relationship. Could you elaborate more on how `LogStoreEnabled` is different from `ConsistentOldValue` and why we need this?

Reply:
`LogStoreEnabled ` has the same op consistency requirement as `ConsistentOldValue`. The difference is that, for `ConsistentOldValue`, in release mode that has no old value sanity check, the behavior of `ConsistentOldValue` is the same as `Inconsistent`, which simply writes and stores only the new value to state store, but for `LogStoreEnabled`, we will also write and upload the old value to state store.

The old value is written and uploaded to state store so that it can be combined with the new value to replay the change log. The change log will be used in partial checkpoint and subscription.